samples/csharp/getting-started/BinaryClassification_UrlClassification/README.md

```diff
@@ -0,0 +1,143 @@
+# URL Classification
+
+| ML.NET version | API type          | Status                        | App Type    | Data type | Scenario            | ML Task                   | Algorithms                  |
+|----------------|-------------------|-------------------------------|-------------|-----------|---------------------|---------------------------|-----------------------------|
+| v1.0.0           | Dynamic API | Up-to-date | Console app | .txt files | URL classification | Binary classification | FieldAwareFactorizationMachine |
+
+In this introductory sample, you'll see how to use [ML.NET](https://www.microsoft.com/net/learn/apps/machine-learning-and-ai/ml-dotnet) to predict whether an URL is malicious or not. In the world of machine learning, this type of classification is known as **binary classification**.
+
+## Dataset
+The dataset used is this: [URL Reputation Data Set] (https://archive.ics.uci.edu/ml/datasets/URL+Reputation)
+This dastaset is collection of 120-days data which contains 2.3 million records and 3.2 million features.
+
+* The first column is Label column where
+  - +1 corresponds to Malicious URL
+  - -1 corresponds to Benign URL
+* Remaining columns are features which are arranged in [sparse matrix](https://en.wikipedia.org/wiki/Sparse_matrix) format. 
+
+## Problem
```

Comment:
Can you highlight in the README that the problem is most of all to deal with a dataset with many columns with sparse data (millions of columns per row, however only around 200 columns have data, per row).

In this case, the trainer and ML task are not really the highlights but working with a large dataset